scss/login.scss

```diff
@@ -0,0 +1,48 @@
+.login-container {
+  background-color: $wild-sand;
+  border-top: 8px solid $cerulean;
+  height: 400px;
+  padding: 20px 24px 24px;
+}
+
+.data-label {
+  align-self: flex-start;
+  font-size: 16px;
+  height: 22px;
+  margin: 16px 0 8px;
+  width: 100%;
+}
+
+.data-input {
+  border-radius: 5px;
+  height: 32px;
+  width: 252px;
+}
+
+.login-btn {
+  background-color: $earls-green;
+  color: $white;
+  margin-top: 24px;
+}
+
+.sign-up-btn {
+  border: 2px solid $earls-green;
+  color: $earls-green;
+}
+
+.generic-btn {
+  border-radius: 5px;
+  font-size: 16px;
+  height: 32px;
+  width: 100%
+}
```

Comment:
If this a generic-btn class it should go in a more global file

Reply:
It is generic only for this view. These are just like "common styles" that both buttons have, so to not repeat them in both button classes, I made this "common" class with the common styles, and the specific styles of each button are in the correspondent class of each one. I don't think this should go in another file, unless there's another view in the project that has also these common styles, for now "the common styles" are just in this view. Do you think another class name would be clearer for this?

Reply:
There are a couple of things regarding the `generic` point of view:

- Remember to always write the generics *before* the specifics, so the way that CSS works, the override will work as expected. For example in this case, if the `login-btn` changes its `border-radius`, it won't work at all because of the `cascading` approach of CSS
- If you are writing `generics`, you should be doing `templates` or `mixins`. Templates work as template classes so you can `extend` from them. For example:
```scss
%button {
  border-radius: 5px;
  font-size: 16px;
  height: 32px;
  width: 100%
}

.login-btn {
  @extend %button;
  ...
}
```
And `mixins` are like function builders where you can do a lot of stuff (check https://sass-lang.com/documentation/at-rules/mixin)
- If you are writing a generics such as buttons, I recommend to put it in a dedicated file under the `scss` folder, that way every component will make use of them